.github/workflows/test.yaml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest & Mac Os
```

Comment:
There is no need for a mac runner here. Please remove this change.

Reply:
Not to mention, I don't believe that's how the flag works https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on